<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="26" passed="17" failed="9" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-02-20T01:42:40 EET" name="Surefire suite" finished-at="2024-02-20T01:43:11 EET" duration-ms="30341">
    <groups>
    </groups>
    <test started-at="2024-02-20T01:42:40 EET" name="Surefire test" finished-at="2024-02-20T01:43:11 EET" duration-ms="30341">
      <class name="login.LoginTests">
        <test-method is-config="true" signature="setUp()[pri:0, instance:login.LoginTests@505fc5a4]" started-at="2024-02-20T01:42:58 EET" name="setUp" finished-at="2024-02-20T01:43:02 EET" duration-ms="4290" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method is-config="true" signature="testlogout()[pri:0, instance:login.LoginTests@505fc5a4]" started-at="2024-02-20T01:43:02 EET" name="testlogout" finished-at="2024-02-20T01:43:02 EET" duration-ms="94" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testlogout -->
        <test-method signature="testLockedUserLogin(java.lang.String)[pri:0, instance:login.LoginTests@505fc5a4]" started-at="2024-02-20T01:43:02 EET" name="testLockedUserLogin" data-provider="InvalidUsers" finished-at="2024-02-20T01:43:03 EET" duration-ms="636" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[locked_out_user]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLockedUserLogin -->
        <test-method is-config="true" signature="checkPerformance(org.testng.ITestResult,java.lang.reflect.Method)[pri:0, instance:login.LoginTests@505fc5a4]" started-at="2024-02-20T01:43:03 EET" name="checkPerformance" finished-at="2024-02-20T01:43:03 EET" duration-ms="3" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=testLockedUserLogin status=SUCCESS method=LoginTests.testLockedUserLogin(java.lang.String)[pri:0, instance:login.LoginTests@505fc5a4] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void login.LoginTests.testLockedUserLogin(java.lang.String)]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkPerformance -->
        <test-method is-config="true" signature="testlogout()[pri:0, instance:login.LoginTests@505fc5a4]" started-at="2024-02-20T01:43:03 EET" name="testlogout" finished-at="2024-02-20T01:43:03 EET" duration-ms="345" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testlogout -->
        <test-method signature="testUserLogin(java.lang.String)[pri:0, instance:login.LoginTests@505fc5a4]" started-at="2024-02-20T01:43:03 EET" name="testUserLogin" data-provider="Users" finished-at="2024-02-20T01:43:04 EET" duration-ms="500" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[standard_user]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testUserLogin -->
        <test-method is-config="true" signature="checkPerformance(org.testng.ITestResult,java.lang.reflect.Method)[pri:0, instance:login.LoginTests@505fc5a4]" started-at="2024-02-20T01:43:04 EET" name="checkPerformance" finished-at="2024-02-20T01:43:04 EET" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=testUserLogin status=SUCCESS method=LoginTests.testUserLogin(java.lang.String)[pri:0, instance:login.LoginTests@505fc5a4] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void login.LoginTests.testUserLogin(java.lang.String) throws java.lang.InterruptedException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkPerformance -->
        <test-method is-config="true" signature="testlogout()[pri:0, instance:login.LoginTests@505fc5a4]" started-at="2024-02-20T01:43:04 EET" name="testlogout" finished-at="2024-02-20T01:43:04 EET" duration-ms="59" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testlogout -->
        <test-method signature="testUserLogin(java.lang.String)[pri:0, instance:login.LoginTests@505fc5a4]" started-at="2024-02-20T01:43:04 EET" name="testUserLogin" data-provider="Users" finished-at="2024-02-20T01:43:04 EET" duration-ms="581" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[visual_user]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testUserLogin -->
        <test-method is-config="true" signature="checkPerformance(org.testng.ITestResult,java.lang.reflect.Method)[pri:0, instance:login.LoginTests@505fc5a4]" started-at="2024-02-20T01:43:04 EET" name="checkPerformance" finished-at="2024-02-20T01:43:04 EET" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=testUserLogin status=SUCCESS method=LoginTests.testUserLogin(java.lang.String)[pri:0, instance:login.LoginTests@505fc5a4] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void login.LoginTests.testUserLogin(java.lang.String) throws java.lang.InterruptedException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkPerformance -->
        <test-method is-config="true" signature="testlogout()[pri:0, instance:login.LoginTests@505fc5a4]" started-at="2024-02-20T01:43:04 EET" name="testlogout" finished-at="2024-02-20T01:43:05 EET" duration-ms="52" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testlogout -->
        <test-method signature="testUserLogin(java.lang.String)[pri:0, instance:login.LoginTests@505fc5a4]" started-at="2024-02-20T01:43:05 EET" name="testUserLogin" data-provider="Users" finished-at="2024-02-20T01:43:05 EET" duration-ms="423" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[problem_user]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testUserLogin -->
        <test-method is-config="true" signature="checkPerformance(org.testng.ITestResult,java.lang.reflect.Method)[pri:0, instance:login.LoginTests@505fc5a4]" started-at="2024-02-20T01:43:05 EET" name="checkPerformance" finished-at="2024-02-20T01:43:05 EET" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=testUserLogin status=SUCCESS method=LoginTests.testUserLogin(java.lang.String)[pri:0, instance:login.LoginTests@505fc5a4] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void login.LoginTests.testUserLogin(java.lang.String) throws java.lang.InterruptedException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkPerformance -->
        <test-method is-config="true" signature="testlogout()[pri:0, instance:login.LoginTests@505fc5a4]" started-at="2024-02-20T01:43:05 EET" name="testlogout" finished-at="2024-02-20T01:43:05 EET" duration-ms="57" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testlogout -->
        <test-method signature="testUserLogin(java.lang.String)[pri:0, instance:login.LoginTests@505fc5a4]" started-at="2024-02-20T01:43:05 EET" name="testUserLogin" data-provider="Users" finished-at="2024-02-20T01:43:05 EET" duration-ms="407" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[error_user]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testUserLogin -->
        <test-method is-config="true" signature="checkPerformance(org.testng.ITestResult,java.lang.reflect.Method)[pri:0, instance:login.LoginTests@505fc5a4]" started-at="2024-02-20T01:43:05 EET" name="checkPerformance" finished-at="2024-02-20T01:43:05 EET" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=testUserLogin status=SUCCESS method=LoginTests.testUserLogin(java.lang.String)[pri:0, instance:login.LoginTests@505fc5a4] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void login.LoginTests.testUserLogin(java.lang.String) throws java.lang.InterruptedException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkPerformance -->
        <test-method is-config="true" signature="testlogout()[pri:0, instance:login.LoginTests@505fc5a4]" started-at="2024-02-20T01:43:05 EET" name="testlogout" finished-at="2024-02-20T01:43:05 EET" duration-ms="46" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testlogout -->
        <test-method signature="testUserLogin(java.lang.String)[pri:0, instance:login.LoginTests@505fc5a4]" started-at="2024-02-20T01:43:05 EET" name="testUserLogin" data-provider="Users" finished-at="2024-02-20T01:43:11 EET" duration-ms="5356" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[performance_glitch_user]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testUserLogin -->
        <test-method is-config="true" signature="checkPerformance(org.testng.ITestResult,java.lang.reflect.Method)[pri:0, instance:login.LoginTests@505fc5a4]" started-at="2024-02-20T01:43:11 EET" name="checkPerformance" finished-at="2024-02-20T01:43:11 EET" duration-ms="22" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=testUserLogin status=SUCCESS method=LoginTests.testUserLogin(java.lang.String)[pri:0, instance:login.LoginTests@505fc5a4] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void login.LoginTests.testUserLogin(java.lang.String) throws java.lang.InterruptedException]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Time taken for testUserLogin to run test is: 5356 miliiseconds while expected time should be 2000 miliiseconds ]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Time taken for testUserLogin to run test is: 5356 miliiseconds while expected time should be 2000 miliiseconds 
at login.LoginTests.checkPerformance(LoginTests.java:48)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:393)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:326)
at org.testng.internal.invokers.TestInvoker.runConfigMethods(TestInvoker.java:810)
at org.testng.internal.invokers.TestInvoker.runAfterConfigurations(TestInvoker.java:779)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:755)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:155)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:169)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:88)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:137)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkPerformance -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:login.LoginTests@505fc5a4]" started-at="2024-02-20T01:43:11 EET" name="tearDown" finished-at="2024-02-20T01:43:11 EET" duration-ms="0" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- login.LoginTests -->
      <class name="inventory.InventoryTests">
        <test-method is-config="true" signature="setUp()[pri:0, instance:inventory.InventoryTests@3d6f0054]" started-at="2024-02-20T01:42:41 EET" name="setUp" finished-at="2024-02-20T01:42:46 EET" duration-ms="5954" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="testUserIsAbleToClickAddToCard(java.lang.String,java.lang.String)[pri:1, instance:inventory.InventoryTests@3d6f0054]" started-at="2024-02-20T01:42:47 EET" name="testUserIsAbleToClickAddToCard" data-provider="InventorySession" finished-at="2024-02-20T01:42:48 EET" duration-ms="1118" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[session-username]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[standard_user]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testUserIsAbleToClickAddToCard -->
        <test-method signature="testUserIsAbleToClickAddToCard(java.lang.String,java.lang.String)[pri:1, instance:inventory.InventoryTests@3d6f0054]" started-at="2024-02-20T01:42:48 EET" name="testUserIsAbleToClickAddToCard" data-provider="InventorySession" finished-at="2024-02-20T01:42:48 EET" duration-ms="759" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[session-username]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[visual_user]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testUserIsAbleToClickAddToCard -->
        <test-method signature="testUserIsAbleToClickAddToCard(java.lang.String,java.lang.String)[pri:1, instance:inventory.InventoryTests@3d6f0054]" started-at="2024-02-20T01:42:48 EET" name="testUserIsAbleToClickAddToCard" data-provider="InventorySession" finished-at="2024-02-20T01:42:49 EET" duration-ms="521" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[session-username]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[problem_user]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Add/Remove Button is not working]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Add/Remove Button is not working
at inventory.InventoryTests.testUserIsAbleToClickAddToCard(InventoryTests.java:21)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:155)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:169)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:88)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:137)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testUserIsAbleToClickAddToCard -->
        <test-method signature="testUserIsAbleToClickAddToCard(java.lang.String,java.lang.String)[pri:1, instance:inventory.InventoryTests@3d6f0054]" started-at="2024-02-20T01:42:49 EET" name="testUserIsAbleToClickAddToCard" data-provider="InventorySession" finished-at="2024-02-20T01:42:49 EET" duration-ms="464" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[session-username]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[error_user]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Add/Remove Button is not working]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Add/Remove Button is not working
at inventory.InventoryTests.testUserIsAbleToClickAddToCard(InventoryTests.java:21)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:155)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:169)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:88)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:137)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testUserIsAbleToClickAddToCard -->
        <test-method signature="testItemAddedFromCartContainer(java.lang.String,java.lang.String)[pri:2, instance:inventory.InventoryTests@3d6f0054]" started-at="2024-02-20T01:42:49 EET" name="testItemAddedFromCartContainer" data-provider="InventorySession" finished-at="2024-02-20T01:42:50 EET" duration-ms="426" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[session-username]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[standard_user]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testItemAddedFromCartContainer -->
        <test-method signature="testItemAddedFromCartContainer(java.lang.String,java.lang.String)[pri:2, instance:inventory.InventoryTests@3d6f0054]" started-at="2024-02-20T01:42:50 EET" name="testItemAddedFromCartContainer" data-provider="InventorySession" finished-at="2024-02-20T01:42:50 EET" duration-ms="464" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[session-username]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[visual_user]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testItemAddedFromCartContainer -->
        <test-method signature="testItemAddedFromCartContainer(java.lang.String,java.lang.String)[pri:2, instance:inventory.InventoryTests@3d6f0054]" started-at="2024-02-20T01:42:50 EET" name="testItemAddedFromCartContainer" data-provider="InventorySession" finished-at="2024-02-20T01:42:51 EET" duration-ms="366" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[session-username]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[problem_user]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testItemAddedFromCartContainer -->
        <test-method signature="testItemAddedFromCartContainer(java.lang.String,java.lang.String)[pri:2, instance:inventory.InventoryTests@3d6f0054]" started-at="2024-02-20T01:42:51 EET" name="testItemAddedFromCartContainer" data-provider="InventorySession" finished-at="2024-02-20T01:42:51 EET" duration-ms="375" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[session-username]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[error_user]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testItemAddedFromCartContainer -->
        <test-method signature="testResetApp(java.lang.String,java.lang.String)[pri:3, instance:inventory.InventoryTests@3d6f0054]" started-at="2024-02-20T01:42:51 EET" name="testResetApp" data-provider="InventorySession" finished-at="2024-02-20T01:42:52 EET" duration-ms="594" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[session-username]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[standard_user]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[App is not reset to its original state]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: App is not reset to its original state
at inventory.InventoryTests.testResetApp(InventoryTests.java:51)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:155)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:169)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:88)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:137)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testResetApp -->
        <test-method signature="testResetApp(java.lang.String,java.lang.String)[pri:3, instance:inventory.InventoryTests@3d6f0054]" started-at="2024-02-20T01:42:52 EET" name="testResetApp" data-provider="InventorySession" finished-at="2024-02-20T01:42:52 EET" duration-ms="505" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[session-username]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[visual_user]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[App is not reset to its original state]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: App is not reset to its original state
at inventory.InventoryTests.testResetApp(InventoryTests.java:51)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:155)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:169)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:88)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:137)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testResetApp -->
        <test-method signature="testResetApp(java.lang.String,java.lang.String)[pri:3, instance:inventory.InventoryTests@3d6f0054]" started-at="2024-02-20T01:42:52 EET" name="testResetApp" data-provider="InventorySession" finished-at="2024-02-20T01:42:53 EET" duration-ms="488" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[session-username]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[problem_user]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[App is not reset to its original state]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: App is not reset to its original state
at inventory.InventoryTests.testResetApp(InventoryTests.java:51)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:155)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:169)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:88)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:137)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testResetApp -->
        <test-method signature="testResetApp(java.lang.String,java.lang.String)[pri:3, instance:inventory.InventoryTests@3d6f0054]" started-at="2024-02-20T01:42:53 EET" name="testResetApp" data-provider="InventorySession" finished-at="2024-02-20T01:42:53 EET" duration-ms="503" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[session-username]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[error_user]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[App is not reset to its original state]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: App is not reset to its original state
at inventory.InventoryTests.testResetApp(InventoryTests.java:51)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:155)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:169)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:88)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:137)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testResetApp -->
        <test-method signature="testSort(java.lang.String,java.lang.String)[pri:4, instance:inventory.InventoryTests@3d6f0054]" started-at="2024-02-20T01:42:53 EET" name="testSort" data-provider="InventorySession" finished-at="2024-02-20T01:42:54 EET" duration-ms="472" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[session-username]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[standard_user]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSort -->
        <test-method signature="testSort(java.lang.String,java.lang.String)[pri:4, instance:inventory.InventoryTests@3d6f0054]" started-at="2024-02-20T01:42:54 EET" name="testSort" data-provider="InventorySession" finished-at="2024-02-20T01:42:54 EET" duration-ms="428" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[session-username]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[visual_user]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSort -->
        <test-method signature="testSort(java.lang.String,java.lang.String)[pri:4, instance:inventory.InventoryTests@3d6f0054]" started-at="2024-02-20T01:42:54 EET" name="testSort" data-provider="InventorySession" finished-at="2024-02-20T01:42:54 EET" duration-ms="411" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[session-username]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[problem_user]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Sort Filter is not working]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Sort Filter is not working
at inventory.InventoryTests.testSort(InventoryTests.java:63)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:155)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:169)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:88)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:137)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSort -->
        <test-method signature="testSort(java.lang.String,java.lang.String)[pri:4, instance:inventory.InventoryTests@3d6f0054]" started-at="2024-02-20T01:42:54 EET" name="testSort" data-provider="InventorySession" finished-at="2024-02-20T01:42:55 EET" duration-ms="365" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[session-username]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[error_user]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Sort dropdown is broken]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Sort dropdown is broken
at inventory.InventoryTests.testSort(InventoryTests.java:65)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:155)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:169)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:88)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:137)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSort -->
        <test-method signature="testCartStyle(java.lang.String,java.lang.String)[pri:5, instance:inventory.InventoryTests@3d6f0054]" started-at="2024-02-20T01:42:55 EET" name="testCartStyle" data-provider="InventorySession" finished-at="2024-02-20T01:42:55 EET" duration-ms="270" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[session-username]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[standard_user]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCartStyle -->
        <test-method signature="testCartStyle(java.lang.String,java.lang.String)[pri:5, instance:inventory.InventoryTests@3d6f0054]" started-at="2024-02-20T01:42:55 EET" name="testCartStyle" data-provider="InventorySession" finished-at="2024-02-20T01:42:55 EET" duration-ms="214" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[session-username]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[visual_user]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Css style is not correct]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Css style is not correct
at inventory.InventoryTests.testCartStyle(InventoryTests.java:76)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:155)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:169)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:88)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:137)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCartStyle -->
        <test-method signature="testCartStyle(java.lang.String,java.lang.String)[pri:5, instance:inventory.InventoryTests@3d6f0054]" started-at="2024-02-20T01:42:55 EET" name="testCartStyle" data-provider="InventorySession" finished-at="2024-02-20T01:42:56 EET" duration-ms="220" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[session-username]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[problem_user]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCartStyle -->
        <test-method signature="testCartStyle(java.lang.String,java.lang.String)[pri:5, instance:inventory.InventoryTests@3d6f0054]" started-at="2024-02-20T01:42:56 EET" name="testCartStyle" data-provider="InventorySession" finished-at="2024-02-20T01:42:56 EET" duration-ms="240" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[session-username]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[error_user]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCartStyle -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:inventory.InventoryTests@3d6f0054]" started-at="2024-02-20T01:42:56 EET" name="tearDown" finished-at="2024-02-20T01:42:58 EET" duration-ms="2187" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- inventory.InventoryTests -->
    </test> <!-- Surefire test -->
  </suite> <!-- Surefire suite -->
</testng-results>
